lib/jsonapi/serializable.rb

```diff
@@ -1,3 +1,17 @@
 require 'jsonapi/serializable/error'
 require 'jsonapi/serializable/resource'
 require 'jsonapi/serializable/renderer'
+
+module JSONAPI
+  module_function
+
+  # @see JSONAPI::Serializable::SuccessRenderer#render
+  def serialize(resources, options = {})
+    JSONAPI::Serializable::SuccessRenderer.new.render(resources, options)
```

Comment:
I think there is no reason to create a new instance of `(Success|Error)Renderer` each time `#serialize` (and `#serialize_error`) is called.